qutebrowser/browser/network/qutescheme.py

```diff
@@ -153,7 +153,7 @@ def qute_help(win_id, request):
     """Handler for qute:help. Return HTML content as bytes."""
     try:
         utils.read_file('html/doc/index.html')
-    except FileNotFoundError:
+    except (FileNotFoundError, NotADirectoryError, OSError, BlockingIOError):
```

Comment:
This seems like an unrelated change from #763 - can you remove it from the PR please? :smile: